appcenter/AppCenter.js

```diff
@@ -102,6 +104,17 @@ const AppCenter = {
 
     getSdkVersion() {
         return PackageJson.version;
+    },
+
+    setAuthTokenListener(listener) {
+        eventEmitter.removeAllListeners(onAcquireAuthTokenEvent);
+        if (listener && listener.acquireAuthToken) {
```

Comment:
This contradicts typescript definition line 43. The callback is optional here.

Reply:
As discussed, if we are keeping required in TS, we should emit a AppCenterLog.error if listener is not null but callback missing.

Reply:
Bump